src/Adapter/Cart/CommandHandler/AddProductToCartHandler.php

```diff
@@ -73,11 +73,14 @@ public function handle(AddProductToCartCommand $command): void
         $customizationId = null;
```

Comment:
Now the value of `$customizationId` remains null, yet this variable is passed in `$cart->getProductQuantity` and `$this->updateProductQuantityInCartHandler->handle()` just below.

Is it ok that this value will be passed as null all the time ?

Reply:
@matthieu-rolland yes, an optional customization not filled is the same as no customization at all ;)

Reply:
alright ! in that case maybe we can drop the `$customizationId`, since its value is always null and never changes ? (and pass null as a parameter instead below)

Reply:
Or maybe, if `$customizationIdVO` is set, we should put `$customizationIdVO->getValue()` as a parameter instead of `$customizationId` that is always null (in the two functions below) ?

Reply:
Oh! Nice catch! I did something stupid here 😅 I used `$combinationId` as `$customizationId`!